toolbox/cijoe-pkg-xnvme/src/cijoe/xnvme/tests/logic/test_xnvme_tests_ioworker.py

```diff
@@ -0,0 +1,32 @@
+"""
+New addition
+"""
+import pytest
+from cijoe.xnvme.tests.conftest import xnvme_device_driver as device
+from cijoe.xnvme.tests.conftest import xnvme_setup
+
+
+@pytest.mark.parametrize(
+    "device,be_opts,cli_args",
+    xnvme_setup(labels=["dev"], opts=["be", "sync", "async", "admin"]),
+    indirect=["device"],
+)
+def test_verify(cijoe, device, be_opts, cli_args):
+
+    if be_opts["async"] in ["libaio"]:
+        pytest.skip(reason="FIXME: [async=libaio] needs investigation")
+
+    err, _ = cijoe.run(f"xnvme_tests_ioworker verify {cli_args}")
+    assert not err
+
+
+@pytest.mark.parametrize(
+    "device,be_opts,cli_args",
+    xnvme_setup(labels=["bdev"], opts=["be", "sync", "async", "admin"]),
+    indirect=["device"],
+)
+@pytest.mark.skip(reason="FIXME: --direct=1 needs investigation")
```

Comment:
It is not a regression, since we haven't done this before.
So I would rather go ahead and get the infra in and then use it to speed of the fixing of this and other issues.